icu4c/source/common/dictbe.cpp

```diff
@@ -1385,6 +1406,26 @@ CjkBreakEngine::divideUpDictionaryRange( UText *inText,
     // inputMap goes out of scope
     return numBreaks;
 }
+
+void CjkBreakEngine::loadJpParticleAndAuxVerbs() {
```

Comment:
Where do you call this function, I cannot findit.

Reply:
Resolved. It's in the constructor.
(Something lost as fixing conflict)